site/source/user-manual/backups/backup-setup/backup-windows.rst

```diff
@@ -10,6 +10,12 @@ Windows LAN Shared Folder
 
 Use this guide to setup a LAN Shared Folder on your Windows machine (or attached drive), such that you may create encrypted, private backups of all your Embassy data.
 
+Video Guide:
```

Comment:
No need to comment on video vs text guides, simply add the video link at the top of the text guide, underneath the heading, as done on other pages with videos.  We want to keep uniformity

Reply:
Fixed